src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md

```diff
@@ -0,0 +1,265 @@
+- Modularization of scripts for collaboration and easier maintenance.
+- Preference to work in your local environment.
+- Integrating testing in CI/CD pipelines.
+
+## Instructions
+
+1. First, you need to have a k6 Cloud account. If you don't have one, sign up [here](https://app.k6.io/account/register) and get 50 cloud tests with the Free Trial.
+
+2. Install k6 using the instructions [here](/getting-started/installation).
+
+3. Authenticate to k6 Cloud from the CLI. Log in using your username and password or your [API token](https://app.k6.io/account/api-token).
+
+    <div class="code-group" data-props='{"labels": ["Log in with username and password", "Log in with the API Token"]}'>
+
+    ```shell
+    $ k6 login cloud
+    ```
+
+    ```shell
+    $ k6 login cloud --token <YOUR_K6_CLOUD_API_TOKEN>
+    ```
+
+    </div>
+
+    `k6 login` stores your API Token in a local config file to authenticate to k6 Cloud when running cloud commands.
+
+
+4. Run your test in the cloud.
+
+   <div class="code-group" data-props='{"labels": ["CLI", "Docker"]}'>
+
+   ```shell
+   $ k6 cloud script.js
+   ```
+
+   ```shell
+   # Note the difference in specifying the `K6_CLOUD_TOKEN` environment variable 
+   # using the `docker run -e` option.
+
+   $ docker run -i -e K6_CLOUD_TOKEN=<API_TOKEN> loadimpact/k6 cloud - <script.js
+
+
+   # When passing the script via stdin there is no way for the containerized k6 process
+   # to get the script filename, which is required for cloud execution. To solve this issue:
+   
+   # a) specify the `options.ext.loadimpact.name` property in the script:
+
+   #   export let options = {
+   #    ext: {
+   #      loadimpact: {
+   #        name: 'My awesome test',
+   #      },
+   #    },
+   #  };
+   
+   # b) mount the local filesystem as a Docker volume:
+
+   $ docker run -i -e ... -v "$PWD/script.js:/script.js" loadimpact/k6 cloud /script.js
+   ```
+
+   </div>
+
+
+6. You'll see k6 print some information and the URL of your test results.
+
+    <div class="code-group" data-props='{"labels": [""]}'>
+
+    ```shell
+             /\      |‾‾|  /‾‾/  /‾/   
+        /\  /  \     |  |_/  /  / /    
+       /  \/    \    |      |  /  ‾‾\  
+      /          \   |  |‾\  \ | (_) | 
+     / __________ \  |__|  \__\ \___/ .io
+
+      execution: cloud
+      script: test.js
+      output: https://app.k6.io/runs/TEST_RUN_ID
+    ```
+
+    </div>
+
+7. Navigate to the URL to check your test results.  When the test is running, the test result page is shown. 
+
+   ![Insights results](/images/cloud-insights-results.png 'k6 Cloud Test Results')
+
+   Learn more about the different test result sections on the [k6 Cloud Results docs](/cloud/analyzing-results/overview).
+
+## Cloud execution options
+
+All the [k6 Options](/using-k6/options), like `--vus` and `--duration` are the same between the `k6 run` and `k6 cloud` commands. k6 aims to run the same script on different execution modes without making any script modifications.
```

Comment:
"**on** different execution modes" => "**in** different execution modes"

Reply:
https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720